web/src/modules/administration/modules/injest-data/components/DataIngestionEditor.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <v-dialog
+    v-model="internalVisible"
+    persistent
```

Comment:
If you find that you need to make a modal persistent, consider using a page instead.
Persistent modals are very confusing for most users who expect the escape key to exit the modal.